Mathlib/Data/Nat/Cast/Order/Basic.lean

```diff
@@ -98,6 +98,8 @@ theorem cast_lt_one : (n : α) < 1 ↔ n = 0 := by
 @[simp, norm_cast]
 theorem cast_le_one : (n : α) ≤ 1 ↔ n ≤ 1 := by rw [← cast_one, cast_le]
 
+@[simp] lemma cast_nonpos : (n : α) ≤ 0 ↔ n ≤ 0 := by norm_cast
```

Comment:
Should we simplify to `n ≤ 0` or `n = 0`?

Reply:
I suggest `n = 0`, especially as that agrees with `cast_lt_one` above.